src/transport.js

```diff
@@ -53,11 +54,18 @@ module.exports = function (swarm) {
       // specific for the transport we are using
       const proxyConn = new Connection()
 
-      next(multiaddrs.shift())
+      each(multiaddrs, next)
```

Comment:
Mmm... that makes sense. 

Does it makes sense to keep all the available connections around? Would it makes sense to switch between connection depending on latency (probably overkill)?

In fairness tho, the first one to come back is probably the fastest one already 🍭